src/languages/en-US.ts

```diff
@@ -2492,6 +2492,28 @@ export default class extends Language {
 				Now get those coins flippin'.`,
 			examples: ['50 heads', '200 tails']
 		}),
+		COMMAND_HIGHERLOWER_DESCRIPTION: 'Play a game of Higher/Lower',
+		COMMAND_HIGHERLOWER_EXTENDED: builder.display('higherlower', {
+			extendedHelp: `Higher Lower is a game of luck. I pick 2 numbers, and you have to guess if the 2nd is **higher** or **lower** then the 1st, using the ⬆ or ⬇ emojis.
```

Comment:
```suggestion
			extendedHelp: `Higher Lower is a game of luck. I will pick a number and you'll have to guess if the next number I pick will be **higher** or **lower** than the current one, using the ⬆ or ⬇ emojis.
```

Reworded a bit, I think this flows better.